pkg/cmd/secret/secret.go

```diff
@@ -15,11 +15,9 @@ func NewCmdSecret(f *cmdutil.Factory) *cobra.Command {
 		Short: "Manage GitHub secrets",
 		Long: heredoc.Doc(`
 			Secrets can be set at the repository, environment, or organization level for use in
-			GitHub Actions. Run "gh help secret set" to learn how to get started.
-		`),
-		Annotations: map[string]string{
```

Comment:
Resolved conflicts with #4675, with this change secrets would no longer just be an Actions command. Let me know if there's a better tag here.